skydb/skydb.go

```diff
@@ -88,6 +87,29 @@ func (db SkyDB) Write(data []byte, dataKey crypto.Hash, rev uint64) error {
 	return nil
 }
 
+// EntropyFromEnv returns the configured value of the SKYDB_ENTROPY environment
+// variable or an error.
+func EntropyFromEnv() (crypto.Hash, error) {
+	var e crypto.Hash
+	eStr := os.Getenv("SKYDB_ENTROPY")
+	if eStr == "" {
+		return e, errors.New("missing or empty SKYDB_ENTROPY environment variable. it needs to contain 32 bytes of base64 encoded entropy.")
+	}
+	eBytes, err := base64.StdEncoding.DecodeString(eStr)
+	if err != nil || len(eBytes) != 32 {
+		return e, fmt.Errorf("invalid SKYDB_ENTROPY environment variable. it needs to contain 32 bytes of base64 encoded entropy. error: %v", err)
+	}
+	copy(e[:], eBytes)
+	return e, nil
+}
+
+// skynetFilePath returns a path that a Skyfile can be uploaded to. The path is
+// based on the provided dataKey.
+func skynetFilePath(dataKey crypto.Hash) (sp skymodules.SiaPath) {
```

Comment:
Is this not called by this MR? Also could this leas to collisions when different pubkeys use the same datakey?